src/ts/ove.ts

```diff
@@ -23,23 +23,40 @@ export function displayPlasmidViewer(): void {
     const filename = $(this).data('href');
     const realName = $(this).data('realName');
 
-    // helper function to convert Blob to File
+    // A Blob() is almost a File(): it's just missing two properties (lastModified and a name)
+    // we also add the optional (mime) type attribute
     function blobToFile(theBlob: Blob, fileName: string): File {
```

Comment:
great variable naming ^^ https://en.wikipedia.org/wiki/The_Blob

Reply:
I did not know that movie. But maybe the author of that line, I copied it from somewhere on the internet.